examples/prt4cmd/src/CMakeLists.txt

```diff
@@ -34,7 +34,7 @@ if(NOT prt_DIR)
 		set(PRT_TC "gcc93")
 	endif()
 
-	set(PRT_VERSION "3.0.8905")
+	set(PRT_VERSION "3.1.9664")
```

Comment:
It looks like it's going to be `9665`, but let's wait until we have confirmation.